src/frontend/parallelization_phase.nit

```diff
@@ -39,10 +39,9 @@ private class ParallelizationPhase
 			toolcontext.error(nmethdef.location, "Syntax error: only a method can be threaded.")
 			return
 		end
-		if nmethdef.n_signature.n_params.length != 0 then
-			toolcontext.error(nmethdef.location, "Syntax error: parametrized method not supported yet.")
-			return
-		end
+
+		#TODO: check for self calls
```

Comment:
There was no plan yet, that's why it isn't done in this version. But I'm for the first option